### PR TITLE
homework-W2 -DB -Kumait 

### PR DESCRIPTION
**Pull request by 29Kumait**

### CODE REVIEW

**Comment by 29Kumait** [comment]
<p class="p1" style="margin: 0px; font-style: normal; font-variant-caps: normal; font-weight: 400; font-stretch: normal; font-size: 17px; line-height: normal; font-size-adjust: none; font-kerning: auto; font-variant-alternates: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-east-asian: normal; font-variant-position: normal; font-variant-emoji: normal; font-feature-settings: normal; font-optical-sizing: auto; font-variation-settings: normal; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-tap-highlight-color: rgba(26, 26, 26, 0.3); -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span class="s1" style="font-family: UICTFontTextStyleBody; font-weight: normal; font-style: normal; font-size: 17px;"> <h3> Commit refactor </h3>  I solved the exercise using the CLI beforehand. Some of the tasks only display table,<span class="Apple-converted-space"> <span class="Apple-converted-space"> </span></span>but I added creating a table to keep it. When I did the exercise again using JavaScript,<span class="Apple-converted-space"> <span class="Apple-converted-space"> </span></span>I encountered some Errors; because of trying to create a new table that was already created in the same database. To fix the problem,<span class="Apple-converted-space"> <span class="Apple-converted-space"> </span></span>I had to do start the exercise on another new database, and then I also had to include the DROP TABLE statement to delete the table that I had created instead of displaying. In addition to other issues like: </span></p><p class="p2" style="margin: 0px; font-style: normal; font-variant-caps: normal; font-weight: 400; font-stretch: normal; font-size: 17px; line-height: normal; font-size-adjust: none; font-kerning: auto; font-variant-alternates: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-east-asian: normal; font-variant-position: normal; font-variant-emoji: normal; font-feature-settings: normal; font-optical-sizing: auto; font-variation-settings: normal; min-height: 24px; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-tap-highlight-color: rgba(26, 26, 26, 0.3); -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span class="s1" style="font-family: UICTFontTextStyleBody; font-weight: normal; font-style: normal; font-size: 17px;"></span><br></p>

 > Error MySQL's restriction on executing multiple queries in a single query

 > Error "Pool is closed" because the connection.end() is called before the queries are executed.


[refactor](https://github.com/HackYourAssignment/databases-class45/pull/13/commits/36741a4e3b1bbf5787a5b1edcfcaf2bbce023998)

File: Week2/w2DB/logfile1st.txt
Comment (29Kumait) [inline Week2/w2DB/logfile1st.txt:1]: In this `commit` the tasks of the homework/assignment are done by the `terminal`.
--
[logfile.txt](https://github.com/HackYourAssignment/databases-class45/pull/13/commits/45a57e54e185c20db436ce08e6c9ae44fdab2fc6#diff-6bddd58232ecb1f0025ef8b9c2ed4067e65e2f1ea63b2bfb0247698f10873637)

**Comment by 29Kumait** [comment]
In this `commit` the tasks of the homework/assignment are done by the `terminal`.
--
[logfile.txt](https://github.com/HackYourAssignment/databases-class45/pull/13/commits/45a57e54e185c20db436ce08e6c9ae44fdab2fc6#diff-6bddd58232ecb1f0025ef8b9c2ed4067e65e2f1ea63b2bfb0247698f10873637)

**Review by anis790615 — CHANGES_REQUESTED** [review]
Sorry, I can't review this. Please, don't commit irrelevant files in the pr. Your requirement is 4 exercises. So, only the exercise files should be in the pr, preferably with a name the same as the name of the exercise. Also don't change the name of the user or the password, 'cause that means that in order for me to test it, i must either manually create the user or change the user and the password in each file to test it locally.

**Comment by 29Kumait** [comment]
[No worry](https://github.com/HackYourAssignment/databases-class45/pull/33)

**Pull request closed without merging**